backend/src/getLogs.py

```diff
@@ -43,10 +97,20 @@
             log.logEvents[QUOTE_LOG](transact)
         elif (transact['logType'] == TRANSACT_LOG):
             log.logEvents[TRANSACT_LOG](transact)
+        
+        #if (processing % 100000):
+        if not (processing % 25000):
```

Comment:
just curious why % 25000 here?

Reply:
I just put 25000 here as the old getLogs would die at around ~150000 transactions processed. so the number just needs to be below that. I'll get rid of the comment above. 

still gotta test it with the 1000 user WL thats on my plate today.